api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -21,16 +34,28 @@
 /**
  * This class represents a list of patientIds.
  */
+@Entity
+@Table(name = "cohort")
+@AttributeOverride(name = "uuid", column = @Column(name = "uuid", length = 38, unique = true))
 public class Cohort extends BaseChangeableOpenmrsData {
-	
+
 	public static final long serialVersionUID = 0L;
-	
+
+	@Id
+	@Column(name = "cohort_id")
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "cohort_id_gen")
+	@SequenceGenerator(name = "cohort_id_gen", sequenceName = "cohort_cohort_id_seq")
 	private Integer cohortId;
-	
+
+	@Column(nullable = false)
 	private String name;
-	
+
+	@Column(nullable = false, length = 1000)
```

Comment:
Why did you add the length when you had specified as null

Reply:
The field can't be null